student_list2/lib/main.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final title = 'Students List';
+
+    return MaterialApp(
+      title: title,
+      home: Scaffold(
+        appBar: AppBar(title: Text(title)),
+        body: ListView(
+          children: new List.generate(100, (index) => new ListTile(
```

Comment:
You don't need to write `new`. Please remove it

Reply:
got it:-)